health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/BreadCrumbNew.js

```
@@ -0,0 +1,140 @@
+      setCrumbsToDisplay([...props.crumbs]);
+    }
+  }, [props.crumbs, props.maxItems, expanded, props.itemsBeforeCollapse, props.itemsAfterCollapse, props?.expandText]);
+
+  function handleRedirect(path) {
+    const host = window.location.origin; // Dynamically get the base URL
+    window.location.href = `${host}${path}`;
+  }
+
+  function isLast(index) {
+    // Filter crumbs to only include those that are displayed
+    let validCrumbs = crumbsToDisplay?.filter((crumb) => crumb?.show === true);
```

Biome

[error] This let declares a variable that is only assigned once.

'validCrumbs' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)